test/android/cmake/AndroidManifest.xml.cmake

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="@ANDROID_MANIFEST_PACKAGE@">
+
+    <!-- OpenGL ES 2.0 -->
+    <uses-feature android:glEsVersion="0x00020000" />
+
+    <!-- Touchscreen support -->
+    <uses-feature
+        android:name="android.hardware.touchscreen"
+        android:required="false" />
+
+    <!-- Game controller support -->
+    <uses-feature
+        android:name="android.hardware.bluetooth"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.gamepad"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.usb.host"
+        android:required="false" />
+
+    <!-- External mouse input events -->
+    <uses-feature
+        android:name="android.hardware.type.pc"
+        android:required="false" />
+
+    <!-- Allow access to the vibrator -->
+    <uses-permission android:name="android.permission.VIBRATE" />
+
+    <!-- Allow access to the microphone -->
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/sdl-test"
+        android:roundIcon="@mipmap/sdl-test_round"
```

Comment:
I've created a svg approximation of [SDL's logo](https://github.com/libsdl-org/SDL/blob/main/Xcode/SDL/pkg-support/sdl_logo.pdf):

![sdl_logo](https://user-images.githubusercontent.com/4138939/235517567-773c674a-10af-4222-abd8-25c481f02dd0.svg)


![SDL_beta](https://user-images.githubusercontent.com/4138939/235517580-df9af453-d321-4bc0-b8de-cb27293bc133.svg)